tests/runtime/test_grpc_server.py

```diff
@@ -365,6 +365,10 @@ def test_train_fake_module_does_not_change_another_instance_model_of_block(
 
     # Train an OtherModule with batch size 100
     stream_type = caikit.interfaces.common.data_model.DataStreamSourceInt
+    training_parameters_json_dict = caikit.core.data_model.json_dict.dict_to_struct(
```

Comment:
can this `caikit.core.data_model.json_dict.dict_to_struct` conversion not happen in the background if the user passes in a dict? 🤔

Reply:
🤔  In all of these tests we're building the grpc request directly, but we don't necessarily have to. We could use the datamodel classes directly instead.

Definitely don't have to swap for this PR but that could make our tests a bit simpler, and then all the server tests would cover the `.to_proto`ing of request messages as well

Reply:
Opened an issue for this as well! https://github.com/caikit/caikit/issues/251